flytekit/extras/tasks/shell.py

```diff
@@ -237,3 +237,145 @@ def execute(self, **kwargs) -> typing.Any:
 
     def post_execute(self, user_params: ExecutionParameters, rval: typing.Any) -> typing.Any:
         return self._config_task_instance.post_execute(user_params, rval)
+
+
+class _RawShellTask(ShellTask):
```

Comment:
```suggestion
class RawShellTask(ShellTask):
```

Reply:
why the `_`

Reply:
I did not think there was too much value in exposing the class as without a template, it doesn't really differ too much from the regular `ShellTask`. I suppose I could add usage examples of supplying your own template into `flytesnacks`